src/bento/caseDetailsData.js

```diff
@@ -176,6 +178,7 @@ export const table2 = {
       dataField: 'uuid',
       header: 'UUID',
       display: false,
+      primary: true,
```

Comment:
why didn't we use this instead of primary key index?

Reply:
Column setting will be redefined from Bento-tools's function getColumns. 
The attribute primary was not included in the getColumns function.  Even set this attribute here, will not be used.  Change the getColumns from bento-tools is challenging at the point and we need some design thinking of how to improve the getColumns and getOption functions to be more flexible.